_posts/2018-12-10-snowplow-javascript-tracker-2.10.0-released/index.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "Snowplow JavaScript Tracker 2.10.0 released with global contexts"
+title-short: Snowplow JavaScript Tracker 2.10.0
+tags: [snowplow, javascript, global, contexts, guard, error]
+author: Mike
+category: Releases
+permalink: /blog/2018/12/10/snowplow-javascript-tracker-2.10.0-released-with-global-contexts/
+discourse: true
+---
+
+We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js-tracker]. [Version 2.10.0][2.10.0-tag] introduces global contexts, a set of powerful features for working with contexts. Also included are: more context features for link and form tracking, a new option to prevent exceptions from surfacing, and many other under-the-hood improvements, updates, and bugfixes.
+
+Read on below the fold for:
+
+1. [Global contexts](#global-contexts)
+2. [Error-handling](#error-supression)
+3. [Dynamic contexts for link and form tracking](#link-form-tracking)
+4. [Updates and bug fixes](#updates)
+5. [Upgrading](#upgrade)
+6. [Documentation and help](#doc)
+
+<!--more-->
+
+<h2 id="global-contexts">1. Global contexts</h2>
```

Comment:
Push back if there are good reasons why we don't do this... 

Thinking about the reader who hasn't been using the trackers extensively but is reading blog posts, it feels like this section could do with just one more sentence on what contexts are used for to contextualise (sorry, couldn't think of another word) the rest of the post.

For the user who is using the trackers extensively I think we could pull out what this _means_ for them. I.e. less code, less prone to mistakes.